javascript-review.js

```diff
@@ -131,30 +179,77 @@
 // b. 
 // What does this function return?
 
+const functionReturn = "This function returns a keyless object labeling the persons name and running the funtion sayHi() which somply prints the word 'hello'."
```

Comment:
the object isn't keyless, it still has name and sayHi as keys, everything else in this answer is correct